VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -21,22 +21,26 @@
     private Observer mObserver;
 
     IEdxEnvironment environment;
+    UserAPI api;
```

Comment:
I think I tried that when I first did this but ended up getting a null injections :/